include/nbl/asset/metadata/CGLTFMetadata.h

```diff
@@ -31,6 +28,29 @@ class CGLTFMetadata final : public IAssetMetadata
             return static_cast<const asset::CGLTFPipelineMetadata*>(found);
         }
 
+        using INSTANCE_ID = uint32_t;
+
+        struct Scene
+        {
+            core::smart_refctd_ptr<asset::ICPUBuffer> instanceIDs;
```

Comment:
doesn't need to be a buffer, but an okay choice

probably SBufferRange could have been better (as to use one ICPUBuffer for all instanceID lists, and remove the `count`)

Reply:
yeah the following idea is better, TODO

Reply:
corrected in following commit